Source/JavaScriptCore/jit/IntrinsicEmitter.cpp

```diff
@@ -73,6 +75,9 @@ bool IntrinsicGetterAccessCase::canEmitIntrinsicGetter(StructureStubInfo& stubIn
         TypeInfo info = structure->typeInfo();
         return info.isObject() && !info.overridesGetPrototype();
     }
+    case WebAssemblyInstanceExportsIntrinsic: {
+        return structure->typeInfo().type() == WebAssemblyInstanceType;
+    }
```

Comment:
nit: don't need the { } since it only contains a single statement.

Reply:
Changed.